### PR TITLE
Configuration File Sync

### PR DESCRIPTION
**Pull request by Wild1145**
This brings the config file back in Sync (Nearly) with what is on the server currently, less any current patches that TFM does not support.

### CODE REVIEW

File: src/main/resources/config.yml [hunk 1]
```diff
@@ -93,58 +93,89 @@ allow:
...
-  - 'n:b:/pp hugeexplosion:You may not use this particle! Please use another.'
-  - 'n:b:/pp barrier:You may not use this particle! Please use another.'
+  - 'n:b:/spawnmob:_'
   - 'n:b:/setjail:_'
-  - 'n:b:/tfbanlist:Use /banlist to see and wipe the ban list'
-  - 'n:b:/tfipbanlist:Use /banlist to see and wipe the ban list'
 
   # Superadmin commands
   - 's:b:/save-all:_'
+  - 's:b:/summon:Do to security reasons, the use of /summon has been disabled.'
```
Comment (ItzLevvie) [inline src/main/resources/config.yml]: This should be ```Due to ...``` instead of ```Do to ...``` 

The same thing also goes for line 129

Comment (Wild1145) [inline src/main/resources/config.yml]: This is simply the current config on the server. I will advise Mark that the corrections are needed, once he makes the change I will update this PR with the change.

**Comment by mayokek** [comment]
@Wild1145, this PR is conflicting. Might wanna fix that.

**Comment by JeromSar** [comment]
@mayokek It might not be, this PR does not target the development branch.

**Comment by mayokek** [comment]
@JeromSar oh, okay.

**Review by JeromSar — APPROVED** [review]

File: src/main/resources/config.yml [hunk 2]
```diff
@@ -332,7 +370,8 @@ autokick:
 overlord_ips:
   - 176.56.237.227
   - 192.184.86.138
-  - 107.191.101.27
+  - 151.50.73.196
```
Comment (JeromSar) [inline src/main/resources/config.yml:373]: These should not be synced with the server. They should remain the config default

**Review by JeromSar — CHANGES_REQUESTED** [review]

**Comment by Wild1145** [comment]
This was what was sent to me, I am no longer in a position to get the latest version of this now anyway, so would advise this being re-synced with a more up to date version given how old this PR now is.

**Pull request closed without merging**